src/test/kotlin/step4/view/ResultViewTest.kt

```diff
@@ -0,0 +1,21 @@
+package step4.view
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class ResultViewTest {
+    @Test
+    fun `visualize block test`() {
+        val raceResult = ArrayList<Int>()
+
+        raceResult.add(0)
+        raceResult.add(1)
+        raceResult.add(1)
+        raceResult.add(1)
+        raceResult.add(0)
+        raceResult.add(1)
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val raceResult = listOf(0, 1, 1, 1, 0, 1)
```

Reply:
수정하였습니다. 코드가 훨씬 간결해 지네요